libvcx/src/disclosed_proof.rs

```diff
@@ -1238,15 +720,15 @@ mod tests {
             tails_file: None,
             timestamp: None,
         }];
-        assert_eq!(DisclosedProof::build_schemas_json(&credential_ids).unwrap_err().kind(), VcxErrorKind::InvalidSchema);
+        assert_eq!(build_schemas_json_prover(&credential_ids).unwrap_err().kind(), VcxErrorKind::InvalidSchema);
```

Comment:
I think the tests asserting methods from `proof_utils.rs` should be in that file

Reply:
Good point